lib/Constants.js

```diff
@@ -273,3 +273,11 @@ module.exports.Intents = {
     directMessageReactions: 1 << 13,
     directMessageTyping:    1 << 14
 };
+
+module.exports.GuildWidgetStyles = {
+    Shield:  "shield",
+    Banner1: "banner1",
+    Banner2: "banner2",
+    Banner3: "banner3",
+    Banner4: "banner4"
```

Comment:
This capitalization style isn't used anywhere else for enum member names; would recommend using `ALL_CAPS` style for consistency:

```suggestion
    SHIELD:   "shield",
    BANNER_1: "banner1",
    BANNER_2: "banner2",
    BANNER_3: "banner3",
    BANNER_4: "banner4"
```